version-history.md

```diff
@@ -6,6 +6,25 @@ search:
 
 This version history has been driven by what we hear from our wonderful community of users. Please share your comments or ideas in the [issue queue](https://github.com/ddev/ddev/issues). We listen. Or talk to us in any of the [support locations](https://ddev.readthedocs.io/en/stable/#support).
 
+## v1.24 (Released 2024-12)
+
+- [x] Added PHP 8.4. PHP 8.3 is default for new projects
+- [x] NodeJS 22 becomes default
+- [x] Extensive support for matching `mysql`, `mariadb`, and `pq` clients so the client matches the server.
+- [x] MariaDB 11.4 LTS support
+- [x] Traditional Windows ARM64 (Windows Copilot) support
```

Comment:
I'm not sure what 'Windows Copilot' has to do with this?

Reply:
Most of those ARM64 computers of the current generation are labeled "Windows Copilot". However, since that will likely be obsolete before anybody reads this...

```suggestion
- [x] Traditional Windows ARM64 support
```